forge-gui/res/cardsfolder/upcoming/ravener.txt

```diff
@@ -0,0 +1,13 @@
+Name:Ravener
+ManaCost:X G U
+Types:Creature Tyranid
+PT:0/0
+K:Flash
+K:Ravenous
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigFight | TriggerDescription$ Terror from the Deep — When CARDNAME enters the battlefield, target creature attacks target opponent this turn if able.
+A:AB$ Pump | ValidTgts$ Creature | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | ValidTgts$ Opponent | RememberObjects$ ParentTarget,Targeted | StaticAbilities$ MustAttack
```

Comment:
Add `ExileOnMoved$ Battlefield` to help keep GUI/engine clean

Reply:
Wouldn't the effect still hold true for this turn if Ravener was removed?

Reply:
It will trigger on the remembered objects of the effect